e2e/pages/enterFamilyMan/enterFamilyMan.js

```diff
@@ -0,0 +1,10 @@
+const I = actor();
+
+module.exports = {
+
+  caseIdField: '#familyManCaseNumber',
```

Comment:
Let's wrap it in `fields` wrapper object as it is done everywhere else.

```suggestion
  fields: {
    caseId: '#familyManCaseNumber',
  },
```